{"name": "Проверка появления ошибки при авторизации курьера с неверным полем password", "status": "passed", "description": "Создание курьера и его авторизация с неверным полем password, проверка статуса ответа с ошибкой и теrста ошибки в ответе", "start": 1715362818421, "stop": 1715362818710, "uuid": "b1983be3-21ff-461e-a1c3-3ea7e4459644", "historyId": "eb5ede0a873c23694323f8578e505748", "testCaseId": "eb5ede0a873c23694323f8578e505748", "fullName": "tests.test_login_courier.TestLoginCourier#test_fail_with_wrong_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "LAPTOP-DIB55KQC"}, {"name": "thread", "value": "10444-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}